server.js

```diff
@@ -41,13 +41,14 @@ app.use('/assets', express.static(path.join(__dirname, 'assets')));
 //   })
 // );
 
-app.use((req, res, next) => {
-  const { userId } = req.session;
-  if (userId) {
-    res.locals.user = users.find(user => user.id === userId);
-  }
-  next();
-});
+// app.use((req, res, next) => {
```

Comment:
Little harder to tell why theres a conflict here, perhaps other code has been merged while you've been working on this PR that affects this file?

Reply:
This is the sessions code . I commented it out for now, because it was breaking things.